testing/selenium_server.md

```diff
@@ -0,0 +1,76 @@
+## Selenium Server Setup and Testing
+
+Using Selenium Server and a guest virtual machine requires a lot of setup and configuration. The following sections describe the configuration and test execution process when using Selenium Server.
+
+### VirtualBox and Virtual Machine Setup
+
+The following steps describe the process for configuring an Internet Explorer 9 and Windows 7 VM provided by Microsoft.
+
+1. [Download a virtual machine](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) from Microsoft by selecting 'IE9 on Win7' with 'VirtualBox' as the platform.
+
+2. Import the virtual machine with VirtualBox.
+
+3. Configure your VirtualBox preferences host-only adapter.
+    - [Open](/testing/virtualbox_hostonly.png) Host-only Network `vboxnet0` for [configuration](/testing/virtualbox_hostonly_config.png)
+    - Change the IPv4 Address to `10.100.1.1`
+    - Change the IPv4 Network Mask to `255.255.255.0`
+
+4. Change the virtual machine network configuration.
+    - [Adapter 1](/testing/guest_machine_adapter_1.png)
```

Comment:
I think you need to add a `!` to the beginning of these so they will so the images:

```md
![Adapter 1](/testing/guest_machine_adapter_1.png)
```